doc/api/packages.md

```diff
@@ -871,6 +861,30 @@ _npm_ registry requires a name that satisfies
 The `"name"` field can be used in addition to the [`"exports"`][] field to
 [self-reference][] a package using its name.
 
+### `"main"`
+<!-- YAML
+added: v0.4.0
+-->
+
+* Type: {string}
+
+```json
+{
+  "main": "./main.js"
+}
+```
+
+The `"main"` field defines the script that is used when the [package directory
+is loaded via `require()`](modules.md#modules_folders_as_modules). Its value
+is interpreted as a path.
+
+```js
+require('./path/to/directory'); // This resolves to ./path/to/directory/main.js.
+```
+
+When a package has an [`"exports"`][] field, this will take precedence over the
+`"main"` field when importing the package by name.
+
```

Comment:
I think @guybedford deliberately wanted this below `"exports"` because we want people to be preferring `"exports"` over `"main"`.

Reply:
the ordering is changed in https://github.com/nodejs/node/pull/35741

I can update it there

Reply:
I'm not tied to the ordering, it just a suggestion previously since there was no explicit ordering (it not being alphabetical).

Reply:
Maybe the list on L825 can have its ordering stated ("In order of precedence, the possible entries are:") while the headers can be in alphabetical order?